.travis.yml

```diff
@@ -9,14 +9,21 @@ language: go
 go:
   - 1.16.5
 
+cache:
+  directories: # ~/.func-e/versions is cached so that we only re-download once: for TestFuncEInstall
+    - $HOME/.func-e/versions
+    - $HOME/go/pkg/mod
+    - $HOME/go/bin/*-v*
+
 git:
   depth: false  # TRAVIS_COMMIT_RANGE requires full commit history.
 
 before_install: |  # Prevent test build of a documentation or GitHub Actions only change.
-  make check || travis_terminate 1
```

Comment:
although this wouldn't have taken a long time with caching setup, it is still better to abort the build vs spend limited minutes